widgetssdk/src/main/java/com/glia/widgets/chat/domain/AddNewMessagesDividerUseCase.kt

```diff
@@ -0,0 +1,20 @@
+package com.glia.widgets.chat.domain
+
+import com.glia.widgets.chat.model.history.ChatItem
+import com.glia.widgets.chat.model.history.NewMessagesItem
+import com.glia.widgets.core.secureconversations.domain.NO_UNREAD_MESSAGES
+
+internal class AddNewMessagesDividerUseCase(
+    private val findNewMessagesDividerIndexUseCase: FindNewMessagesDividerIndexUseCase
+) {
+    operator fun invoke(messages: MutableList<ChatItem>, unreadMessagesCount: Int): Boolean {
+        val index = findNewMessagesDividerIndexUseCase(messages, unreadMessagesCount)
+
+        if (index != NO_UNREAD_MESSAGES) {
```

Comment:
I'd like to suggest improving the readability. I think there are too many negative statements here: !=, "No" and "Unread". There is an opinion that too many negative statements complicate human understanding. Besides, it looks a bit strange that the index is compared to a variable with a boolean-like name. I would like to suggest renaming `NO_UNREAD_MESSAGES` to `INDEX_OUT_OF_RANGE` or something like this.

Reply:
Agree with you! I also don't like this name but didn't find any valuable.
There is a case when the index is not provided, and the name should reflect this also, so what do you think about `NOT_PROVIDED`?

Reply:
fixed [d9a95cf](https://github.com/salemove/android-sdk-widgets/pull/578/commits/d9a95cf17ddc59bf167997cda61337a5ead43dd6)